src/components/DataSubmissions/DataSubmissionSummary.tsx

```diff
@@ -42,35 +44,63 @@ const StyledSubmissionStatus = styled(Typography)(() => ({
   minHeight: "30px",
 }));
 
-const StyledHistoryButton = styled(Button)(() => ({
+const StyledReviewCommentsButton = styled(Button)(() => ({
   "&.MuiButton-root": {
+    minWidth: "168px",
     marginTop: "16px",
+    padding: "11px 10px",
+    border: "1px solid #B3B3B3",
+    color: "#BE4511",
+    fontFamily: "'Nunito', 'Rubik', sans-serif",
+    fontSize: "16px",
+    fontStyle: "normal",
+    fontWeight: 700,
+    lineHeight: "17px",
+    letterSpacing: "0.32px",
+    "&:hover": {
+      backgroundColor: "#FFF",
+      borderColor: "#B3B3B3",
+      color: "#BE4511",
+    },
+    "&:disabled": {
+      backgroundColor: "#FFF",
+      borderColor: "#B3B3B3",
+      color: "#B1B1B1",
+      fontWeight: 700
+    }
+  }
+}));
+
+const StyledHistoryButton = styled(Button)(() => ({
+  "&.MuiButton-root": {
+    minWidth: "168px",
     marginBottom: "10px",
-    padding: "10px 20px",
+    padding: "11px 20px",
     border: "1px solid #004A80",
```

Comment:
Can you double check if #004A80 is the correct border color? The Figma mockup seems to use the same #B3B3B3 color that the Review Comments button has.

Ours:

<img width="197" alt="Screenshot 2024-02-16 at 9 59 49 AM" src="https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/183a97a0-28fb-436b-9798-8b7a0dd889c1">

Figma:

<img width="197" alt="Screenshot 2024-02-16 at 10 01 44 AM" src="https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/fe5fb5d6-a6ca-40e2-bf36-f6dcb8dfbf00">